package.json

```diff
@@ -4,26 +4,31 @@
   "description": "Superset UI",
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "build": "yarn run build:cjs && yarn run build:esm && yarn run type:dts && yarn run build:assets",
-    "build:cjs": "NODE_ENV=production beemo babel --extensions=\".js,.jsx,.ts,.tsx\" ./src --out-dir lib/ --delete-dir-on-start --minify --workspaces=\"@superset-ui/!(demo|generator-superset)\"",
-    "build:esm": "NODE_ENV=production beemo babel --extensions=\".js,.jsx,.ts,.tsx\" ./src --out-dir esm/ --delete-dir-on-start --esm --minify --workspaces=\"@superset-ui/!(demo|generator-superset)\"",
+    "build": "yarn run babel && yarn run type:dts && yarn run build:assets",
```

Comment:
(not the most important thing but noting that) we can remove all of the `run`s with `yarn`

Reply:
fixed